setup.cfg

```diff
@@ -56,3 +58,6 @@ dev =
 
 [flake8]
 extend-ignore = E203, E501
+
+[mypy-sklearn.*]
+ignore_missing_imports = True
```

Comment:
Remove here and add in `pyproject.toml` instead, in the list that already exists (along with `rdkit`)